src/components/open-position.jsx

```diff
@@ -65,6 +66,23 @@ const getOpenContractsInfo = () => {
   );
 };
 
+const onClickSell = async (contract_id, indicative_price, currency) => {
+  try {
+    const resp = await sendRequest({
+      sell: contract_id,
+      price: indicative_price,
+    });
+    setBannerMessage(
+      `Contract Sold Successfully for ${currency} ${resp.sell.sold_for}`
+    );
+    if (resp.error) {
+      throw new Error(resp.error);
+    }
```

Comment:
shouldnt we consider this error response before setting success banner message?

Reply:
Had the same question :D

Reply:
The success message will not appear until and unless the trade is excecuted successfully
It will throw an error any time the contract cannot be sold for it the their is an error in api response